aws-lc-sys/builder/prebuilt-nasm.bat

```diff
@@ -0,0 +1,21 @@
+@echo off
```

Comment:
What does this script do? Why can't the rust program that call this just find all the nasm files itself?

Reply:
What this script does is imitate the part of NASM during the build. When called, it identifies the NASM object corresponding to the indicated source file, and places it into the output directory. We configure Cmake to treat this script as if it were NASM.